common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4884,6 +4886,23 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("3s")
 
+  val RPC_TIMEOUT_RETRY_WAIT: ConfigEntry[Long] =
+    buildConf("celeborn.rpc.timeoutRetryWait")
```

Comment:
Seems `celeborn.rpc.retryWait` is enough.

Reply:
has been updated~